packages/core/ai_content_platform/vimax/pipelines/script2video.py

```
@@ -0,0 +1,162 @@
+            result.errors.append(str(e))
+
+        return result
+
+    def _load_script(self, path: str) -> Script:
+        """Load script from JSON file."""
+        try:
+            with open(path) as f:
+                data = json.load(f)
+            return Script(**data)
+        except FileNotFoundError:
+            raise FileNotFoundError(f"Script file not found: {path}")
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)